src/uct/ib/rc/base/rc_iface.h

```diff
@@ -505,7 +510,8 @@ uct_rc_iface_put_send_op(uct_rc_iface_send_op_t *op)
 {
     uct_rc_iface_t *iface = op->iface;
 
-    ucs_assert(op->flags & UCT_RC_IFACE_SEND_OP_FLAG_IFACE);
+    ucs_assertv(op->flags == UCT_RC_IFACE_SEND_OP_FLAG_IFACE,
```

Comment:
IMO no need to have assert both when allocating and when releasing. one of them is enough

Reply:
removed from `uct_rc_iface_get_send_op`